### ISSUE TITLE
Population calling | joint calling | ensemble calling

### ISSUE THREAD

**Issue by amizeranschi** [nf-core/sarek#868]
### Description of feature

Hi!

I was trying out the `joint_germline` option, which produces a multi-sample VCF file from haplotypecaller. It would be helpful to have a similar feature for other callers like deepvariant, freebayes and strelka2. 

It could also nice to have support for ensemble calling (from multiple callers), for some use cases where the number of samples is manageable.

One way to achieve this would be to use [bcbio.variation.recall](https://github.com/bcbio/bcbio.variation.recall), which was created by Brad Chapman and [used in bcbio-nextgen](https://bcbio-nextgen.readthedocs.io/en/latest/contents/germline_variants.html?highlight=bcbio.variation.recall#parameters).

Relevant documention from bcbio-nextgen can be found here:

* [population calling](https://bcbio-nextgen.readthedocs.io/en/latest/contents/germline_variants.html?highlight=bcbio.variation.recall#parameters)
* [ensemble calling](https://bcbio-nextgen.readthedocs.io/en/latest/contents/somatic_variants.html?highlight=ensemble#ensemble-variant-calling)

For joint calling, another solution could be to have the variant callers output gVCF files, and then use GATK's GenotypeGVCFs or something similar on those. There was a discussion about this on the deepvariant GitHub page a few years ago: https://github.com/google/deepvariant/issues/142#issuecomment-459440356.

**Comment by amizeranschi** [comment]
Past relevant feature request that I just noticed, but also ties into this: 
https://github.com/nf-core/sarek/issues/453

**Comment by SofiaMAhmed** [comment]
Hi Thanks for this really useful tool.   I used it recently but the joint calling option failed for freebayes.   Have others reported this issue?

**Comment by FriederikeHanssen** [comment]
hi! There is no joint calling currently implemented for freebayes.

**Comment by FriederikeHanssen** [comment]
I don't think we have resources at the moment on the developer side. If you'd like to take a stab at it, we would be very happy to guide you through though :)

**Comment by amizeranschi** [comment]
I was looking into this a while ago, but couldn't figure out any practical way to do it. The bcbio-variation-recall package I mentioned above has long been discontinued. Outside of that, there was [GLNexus](https://github.com/dnanexus-rnd/GLnexus) which looked promising, but that has become discontinued as well a couple of years ago and there are currently some serious issues [like this](https://github.com/dnanexus-rnd/GLnexus/issues/286) that were reported but never solved.
